### PR TITLE
According to issue 245 updating the position of wear custom

### PR DESCRIPTION
**Pull request by a-ayyash**
my Habitica User-ID: 2d5da803-98a0-43b8-ba82-10a31a96fbb1

### CODE REVIEW

**Comment by vIiRuS** [comment]
Looks good! Thanks!

**Comment by a-ayyash** [comment]
^_^

File: HabitRPG/Base.lproj/Main.storyboard [hunk 1]
```diff
@@ -209,7 +209,7 @@
                                         <rect key="frame" x="0.0" y="28" width="600" height="67"/>
                                         <autoresizingMask key="autoresizingMask"/>
                                         <tableViewCellContentView key="contentView" opaque="NO" clipsSubviews="YES" multipleTouchEnabled="YES" contentMode="center" tableViewCell="J05-li-dNN" id="jVr-kD-zKG">
-                                            <rect key="frame" x="0.0" y="0.0" width="600" height="66"/>
+                                            <rect key="frame" x="0.0" y="0.0" width="600" height="66.5"/>
```
Comment (nivl4) [inline HabitRPG/Base.lproj/Main.storyboard]: @a-ayyash @vIiRuS can you help me understand why the .5 change in the story board file? my latest version of xcode will automatically remove the .5 after opening the storyboard file.

Comment (a-ayyash) [inline HabitRPG/Base.lproj/Main.storyboard]: I'll check it out, but I don't remember manually adding floating points.

**Pull request merged**